### DISCUSSION TITLE
pip_setup does not require installed python

### DISCUSSION THREAD

**Discussion by kasium** [renovatebot/renovate#28765] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gerrit

### Please tell us more about your question or problem

According to the [docs](https://docs.renovatebot.com/modules/manager/pip_setup/), pip_setup uses python to parse the setup.py file:
> setup.py files are parsed by calling out to python and then using a mock to detect imported modules.

However, according to https://github.com/renovatebot/renovate/issues/6367 and to the code, a JS based parser is used and so there is no need to have a python installation avaibale. Can you please check if the docs could be updated?

**Comment by rarkins** [comment]
The docs are outdated and that sentence should be removed

**Reply by HonkingGoose** [reply, marked as answer]
Removing the sentence, would delete the file. I think some tests/other build code probably expects us to have a `readme.md` file in each manager's docs folder. So I'm replacing the old text with a short blurb about the JS-based parser instead.

See PR:

- #28766

**Reply by kasium** [reply]
Thanks a lot!
